design/decisions/why-lint-package/Why-flake8.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Why Flake8?"
+description: "Overview and rationale for choosing Flake8 as a code
+linting tool."
+author: "Richard"
+date: 2023-11-27
+categories:
+---
+
+## Context and Problem Statement
```

Comment:
This section is about the problem we have (enforcing a consistent style) and not about any given tool.